hawc/apps/animalv2/models.py

```diff
@@ -0,0 +1,576 @@
+from django.core.validators import MinValueValidator
+from django.db import models
+
+from ..assessment.models import BaseEndpoint, DSSTox
+from ..vocab.models import Term
+from . import constants, managers
+
+"""
+Generate a GraphViz dot file of this model:
+
+    manage.py graph_models animalv2 > /path/to/file_to_create.dot
+
+(see https://django-extensions.readthedocs.io/en/latest/graph_models.html for details)
+
+Make an svg/png out of it (assuming you've installed graphviz):
+
+    dot -Tsvg /path/to/previously_generated_file.dot > converted.svg
+"""
+
+
+class Experiment(models.Model):
+    objects = managers.ExperimentManager()
+
+    # TODO - what does this do?!?!
+    TEXT_CLEANUP_FIELDS = ()
+
+    study = models.ForeignKey(
+        "study.Study", on_delete=models.CASCADE, related_name="v2_experiments"
+    )
+    name = models.CharField(
+        max_length=80,
+        verbose_name="Experiment name",
+        help_text="Short-text used to describe the experiment "
```

Comment:
annoying request; but make all long help texts a single line triple-quoted string; it's easier to manage a single long string that our editors can format instead of manually formatting breaks; this is consistent with epiv2 - https://github.com/shapiromatron/hawc/blob/main/hawc/apps/epiv2/models.py

Reply:
I think in general those were ones I just brought over from animalv1 unchanged; pretty sure I got all of these fixed now, does look nicer that way.